src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsRequestBodyFilter.cs

```diff
@@ -30,31 +32,11 @@ public void Apply(OpenApiRequestBody requestBody, RequestBodyFilterContext conte
             if (parameterInfo != null)
             {
                 ApplyParamTags(requestBody, context, parameterInfo);
-                return;
-            }
-        }
-
-        private void ApplyPropertyTags(OpenApiRequestBody requestBody, RequestBodyFilterContext context, PropertyInfo propertyInfo)
-        {
-            var propertyMemberName = XmlCommentsNodeNameHelper.GetMemberNameForFieldOrProperty(propertyInfo);
-            var propertyNode = _xmlNavigator.SelectSingleNode($"/doc/members/member[@name='{propertyMemberName}']");
-
-            if (propertyNode == null) return;
-
-            var summaryNode = propertyNode.SelectSingleNode("summary");
-            if (summaryNode != null)
-                requestBody.Description = XmlCommentsTextHelper.Humanize(summaryNode.InnerXml);
-
-            var exampleNode = propertyNode.SelectSingleNode("example");
-            if (exampleNode == null) return;
-
-            foreach (var mediaType in requestBody.Content.Values)
-            {
-                mediaType.Example = XmlCommentsExampleHelper.Create(context.SchemaRepository, mediaType.Schema, exampleNode.ToString());
             }
         }
 
-        private void ApplyParamTags(OpenApiRequestBody requestBody, RequestBodyFilterContext context, ParameterInfo parameterInfo)
+        private void ApplyParamTags(OpenApiRequestBody requestBody, RequestBodyFilterContext context,
+            ParameterInfo parameterInfo)
```

Comment:
On a side note, I use ReSharper and I think that is why my code is auto-breaking every `x` number of characters

Reply:
I figured it would be ReSharper at work. If there's a sensible way to configure ReSharper to _not_ do this, feel free to check in whatever configuration that needs as part of this PR.